SU-SWS/ood_giving_site
# Release V2.1.1

# READY FOR REVIEW

# Summary
- Redirects and content update for endowed positions

# Steps to test
1. Navigate to https://deploy-preview-255--adapt-giving.netlify.app/endowed-positions
2. Click on any of the schools and ensure that list functionality all works (search and switch)
3. Navigate to https://deploy-preview-255--adapt-giving.netlify.app/endowed-positions/content/purple-monkey-dishwasher
4. Ensure it redirects to https://deploy-preview-255--adapt-giving.netlify.app/endowed-positions

Review code for kicks.
